site/content/posts/2023-08-09-v0.25.0-impersonation-proxy-with-external-certs.md

```diff
@@ -0,0 +1,424 @@
+---
+title: "Pinniped v0.25.0: With External Certificate Management for the Impersonation Proxy and more"
+slug: v0-25-0-external-cert-mgmt-impersonation-proxy
+date: 2023-08-09
+author: Joshua T. Casey
```

Comment:
Added templating to:
- preserve the primary author in the blog post .md file
- template out multiple authors, when present, in the single post view page.